src/components/data/EntitiesContent.test.ts

```diff
@@ -0,0 +1,400 @@
+import { DeepPartial } from '@terra-ui-packages/core-utils';
+import { act, render, screen, within } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import * as clipboard from 'clipboard-polyfill/text';
```

Comment:
I learned to avoid * imports when possible; it seems like there's a Sonar linter which has this opinion too: https://rules.sonarsource.com/typescript/RSPEC-2208/

So maybe instead of doing a `*` import you could just explicitly import `writeText`?  Also the `clipboard` namespace is nice and descriptive, so you could alias it like:

`import {writeText as clipboardWriteText} from 'clipboard-polyfill/text';`

Also curious if @nawatts has an opinion here on *-imports and the approach.

Reply:
Generally, agreed.

Historically, Terra UI uses this `import * as Namespace from '...'` pattern for a few modules (utils, nav, etc.). I would not mind moving away from that, especially in this case where there's one import used and an alias is as effective as a namespace for clarifying where `writeText` is writing text to.